ros1_bridge/__init__.py

```diff
@@ -11,8 +11,33 @@
 # ROS 1 imports
 import genmsg
 import genmsg.msg_loader
+
+# import catkin_pkg and rospkg which are required by rosmsg
+# and likely only available for Python 2
+try:
+    import catkin_pkg
+except ImportError:
+    from importlib.machinery import SourceFileLoader
+    from subprocess import check_output
+    catkin_pkg_path = check_output(['python2', '-c', 'import catkin_pkg; print(catkin_pkg.__file__)'])
```

Comment:
Unfortunately, I think using just `python2` here is a weak assumption, on my machine:

```
% python2
zsh: command not found: python2
```

I do however have python2.7 and python which point to a python2.x interpreter. Maybe that's ok since this is just a recovery behavior, but I wouldn't assume this will work outside of Ubuntu like distros.